src/diff/children.js

```diff
@@ -156,6 +159,7 @@ export function diffChildren(
 								}
 							}
 							parentDom.insertBefore(newDom, oldDom);
+							nextDom = oldDom;
```

Comment:
Since we are inserting `newDom` before `oldDom`, `newDom.nextSibling` will always be `oldDom` so I'm precalculating this now to save us a DOM read.